src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -607,7 +607,7 @@ getAttributionJson apiOpts ProjectRevision{..} = fossaReq $ do
         baseOpts
           <> "includeDeepDependencies" =: True
           <> "includeHashAndVersionData" =: True
-          <> "includeDownloadUrl" =: True
+          <> "dependencyInfoOptions[]" =: Text.pack "PackageDownloadUrl"
```

Comment:
I know that the linter is saying that I should use `toText` here, but I could not get that to work.

I also tried doing this in a previous commit. Happy to do that if it's better:

```haskell
   let packageDownloadUrl :: String
       packageDownloadUrl = "PackageDownloadUrl"
       opts =
         baseOpts
           <> "includeDeepDependencies" =: True
           <> "includeHashAndVersionData" =: True
           <> "dependencyInfoOptions[]" =: packageDownloadUrl
```